client-js/app/elements/labrad-registry.html

```diff
@@ -6,40 +6,188 @@
+      border: 1px solid #ccc;
+      border-top: none;
+    }  
+    .reg-list { 
+      padding-top: 12px;
+      display: table; 
+      border: 1px solid;
+    }
+    .row { 
+      display: table-row; 
+      padding: 12px 24px;
+      width: 100%;
+    }
+
+    .head-item { 
+      display: table-cell; 
+      padding: 12px 24px;
+      border: 1px solid;
+      border-color: yellow;
+      width: 100%;
+    } 
+    .head-grabber { 
+      display: table-cell; 
+      *padding: 12px 24px;
+      border: 1px solid;
+      border-color: yellow;
+    } 
+
+    tbody {
+      display:table-header-group;
     }    
+
+    .table-header {
+      @apply(--paper-font-body1);
+      color: var(--secondary-text-color);
+      *text-align: middle;
+      border-bottom: 1px solid;
+      border-bottom-color: #e5e5e5;
+      display:table-header-group;
+
+    } 
+
+    paper-dialog.size-position {
+      position: fixed;
+      top: 16p
+      right: 16px;
+    }
+    paper-dialog.size-position {
+      width: 300px;
+      height: 300px;
+    }
+    menu-button.done {
+      color: #0F9D58;
+    }
   </style>
   <template>
-  <div class="list short">
-    <table>
-      <thead>
-        <th>Key/Dir Name</th>
-        <th>Key Value</th>
-      </thead>
-      <tbody>
-        <template is="dom-repeat" items="{{dirs}}">
-          <tr>
-            <td>
-              <a is="app-link" path="{{item.url}}" href="{{item.url}}">
-               <dir-item kind="folder">{{item.name}}</dir-item>
-              </a>
-            </td>
-            <td></td>
-          </tr>
-        </template>
-        <template is="dom-repeat" items="{{keys}}">
-          <tr>
-            <td><dir-item kind="key">{{item.name}}</dir-item></td>
-            <td><span class="paper-font-body1">{{item.value}}</span></td>
-          </tr>
-        </template>
-      </tbody>
-    </table>
+    <div id="reg-menu"> 
+    <section onclick="clickHandler(event)">
+      <menu-button data-dialog="doDelete" name="delete" icon-name="delete" class = "done"></menu-button>
+      <menu-button data-dialog="newFolderDialog" name="new folder" icon-name="folder-open" class = "done"></menu-button>
+      <menu-button data-dialog="copyDialog" name="copy" icon-name="content-copy" class = "done"></menu-button>
+      <menu-button data-dialog="renameDialog" name="rename" icon-name="code" class= "done"></menu-button>
+      <menu-button data-dialog="newKeyDialog" name="new key" icon-name="create"  class = "done"></menu-button>
+     </section>
     </div>
+    <div id="reg-area"> 
+      <table>
+          <tbody class="table-header" is="selectable-table"> 
+            <th></th>
+            <th>Key/Dir Name</th>
+            <th>Key Value</th>
+          </tbody>
+          <tbody is="selectable-table"  id="dirList" attr-for-selected="name" selected="{{selDir}}" on-tap="selectDir">       
+            <template is="dom-repeat" items="{{dirs}}">
+                <tr name="{{item.name}}">
```

Comment:
The indentation levels here are somewhat inconsistent. We should stick to two spaces. You can set this up in eclipse, just be aware that the editor preferences for different languages have to be set individually.